extensions/positron-python/python_files/posit/positron_language_server.py

```diff
@@ -90,11 +90,15 @@ def parse_args() -> argparse.Namespace:
     handlers = ["console"] if args.logfile is None else ["file"]
     logging_config = {
         "loggers": {
-            "": {
+            "PositronIPKernelApp": {
                 "level": args.loglevel,
                 "handlers": handlers,
             },
-            "PositronIPKernelApp": {
+            "Comm": {
+                "level": args.loglevel,
+                "handlers": handlers,
+            },
+            "positron._vendor.pygls.protocol.json_rpc": {
```

Comment:
Would this work to catch all child loggers, and possibly be more future-proof?
```suggestion
            "positron": {
```

Reply:
oh! yes, that should work-- updated!